recipes-ni/docker-functional-tests/docker-functional-tests.bb

```diff
@@ -0,0 +1,37 @@
+SUMMARY = "Functional test of docker container capabilities on NILRT"
+HOMEPAGE = "https://docker.com"
+SECTION = "tests"
+LICENSE = "GPL-2.0-only & GPL-2.0-or-later"
```

Comment:
Why not just MIT?

Reply:
I copied over from another test. Is MIT our preference for new recipes?

Reply:
I'd say so; yeah.